### ISSUE TITLE
Could not install package 'SharpFont.Dependencies 2.5.5'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.6.1', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.

### ISSUE THREAD

**Issue by Meai1** [Robmaister/SharpFont#111]
What's the problem? From reading some other issues apparently 4.x is supposed to work?
This message pops up even when I try to only install SharpFont 4.0.1

I'm on linux

**Comment by Meai1** [comment]
tried building myself on linux, had to do tons of workarounds to make it build and then it still throws dllnotfoudnexceptions with missingmethod crap

**Comment by HinTak** [comment]
Yes, building with mono on linux is broken very badly. Head is only about 20 commits from my last successful build, but I have tried 3 ugly hacks already and it is still not building. The most recent successful linux build is v4.0.1+fixes branch on mine, which is only 20 commits or so behind, if that helps.
U
Yes, there are a lot of disagreeable changes among that 20. I still staying at 3.x. But that's because net2 is a must for me. (v4 dropped net2 compatibility, then it got worse and worse)

**Comment by HinTak** [comment]
FWIW, v4.0.1 still builds with mono 4.8.0 - except note
https://github.com/Robmaister/SharpFont/issues/93

**Comment by ghidosoft** [comment]
I have the same issue but I'm using Windows and Visual Studio 2017.
I tried also with previous .NET frameworks (4, 4.5, 4.5.1, 4.5.2) but I got the same error. I even tried both nuget versions 3.1.0/4.0.1 but without success.

**Comment by ghidosoft** [comment]
I "solved" it by first installing the SharpFont.Dependencies nuget package 2.6.0, then I installed the nuget package SharpFont  4.0.1 and it worked flawlessy.

**Comment by HinTak** [comment]
Yes, things after 4.0.1 got very seriously broken. I have given up trying to build from git (3 ugly hacks and still not building is a bit much), and just keeping and maintaining my own heavily patched v3.1.0 and v4.0.1 branches.

**Comment by Meai1** [comment]
What kind of freetype native library version is required for it to work anyway? The weird name "freetype6.dll" is very confusing, that is nothing like the output I get when I compile freetype myself, I get someething like: freetype.so.2.6.1 or something like that, I dont remember. Doesnt even have the right .dll extension, but on linux I cant use .dll right? Yet I see no .so in the git checkout

**Comment by HinTak** [comment]
You not familiar with mono? Mapping to native libraries from windows dll names are in *.dll.config. SharpFont ships one , note https://github.com/Robmaister/SharpFont/issues/93 with it from v4.0 onwards. On the tipical linux box, there are plenty of examples from various `*-sharp` projects (nearly 40  in mine, at least):

```
$ find /usr/lib/mono* -type f -name '*.dll.config' | wc -l
37
```

**Comment by R030t1** [comment]
May be related to #100.
